Obsidian/ConsoleClient.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Microsoft.Extensions.Logging;
+using Obsidian.API;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Obsidian
+{
+    internal class ConsoleClient : IPlayer
+    {
+        public ConsoleClient(ILogger logger, IServer server)
+        {
+            Logger = logger;
+            Server = server;
+        }
+
+        public string Username => "SERVER";
+
+        public Guid Uuid => Guid.Empty;
+
+        public IServer Server { get; }
+
+        public bool IsOperator => true;
+
+        public Gamemode Gamemode { get => default; set => _ = value; }
+        public Hand MainHand { get => default; set => _ = value; }
+        public PlayerBitMask PlayerBitMask { get => default; set => _ = value; }
+        public bool Sleeping { get => default; set => _ = value; }
+        public bool Sneaking { get => default; set => _ = value; }
+        public bool Sprinting { get => default; set => _ = value; }
+        public bool FlyingWithElytra { get => default; set => _ = value; }
+        public bool InHorseInventory { get => default; set => _ = value; }
+        public short AttackTime { get => default; set => _ = value; }
+        public short DeathTime { get => default; set => _ = value; }
+        public short HurtTime { get => default; set => _ = value; }
+        public short SleepTimer { get => default; set => _ = value; }
+        public short CurrentSlot { get => default; set => _ = value; }
```

Comment:
This looks very odd to me. Perhaps trying to access these properties (and methods) should throw `NotSupportedException` instead?

Reply:
This will force plugin devs to use try/catch blocks in commands which is not good ig

Reply:
Well, they should check the value of `CommandContext.Issuer`, if they allow calling from the console. Perhaps `CommandIssuers.Client` should be the default, when `[IssuerScope]` is omitted? I'm not sure.

Reply:
Exceptions are fine, devs should be aware of what can be done and what not.

Reply:
Hmm this does look odd. I think there should be a new interface e.x ICommandSender for this instead of having ConsoleClient implement IPlayer ^ ^

Reply:
in addition to that, returning `default` may return default values, giving a developer (or their code) a bad impression of what data is actually there. Just throw Exceptions like seb said

Reply:
Adding even more to that, it might be a good idea adding abstractions to the Client class.